### PR TITLE
[plugin.audio.rco] 1.0.1

### PR DESCRIPTION
**Pull request by bosel**
### Description
plugin.audi.rco
Update to deal with site change
### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

**Comment by enen92** [comment]
@bosel `plugin.audio.rco` is in the isengard repo. Did you PR to the wrong branch or are you planning to deprecate/keep different plugin versions depending on the kodi version?

**Comment by bosel** [comment]
I don't keep older kodi versions around to do any testing, so depricate I guess (although this particular update should be fine on earlier versions).

**Comment by enen92** [comment]
Please create a second PR as well to mark the addon that is on the isengard repo as broken. Here you have an example: https://github.com/xbmc/repo-scripts/commit/f65b3e85795eb11a5e0b7034aa8514d974c2b77a

**Comment by enen92** [comment]
So can you please make the PR against the isengard branch instead of Jarvis?

**Comment by bosel** [comment]
I'd prefer just to stick with jarvis, if that's OK

File: plugin.audio.rco/addon.xml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.audio.rco" name="Het Koninklijk Concertgebouworkest" version="1.0.1" provider-name="bosel">
+  <requires>
+      <import addon="xbmc.python" version="2.14.0"/>
```
Comment (enen92) [inline plugin.audio.rco/addon.xml]: please set `xbmc.python` to 2.24.0 so it matches jarvis. See: http://kodi.wiki/view/addon.xml

File: plugin.audio.rco/addon.xml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  </requires>
+  <extension point="xbmc.python.pluginsource" library="rco.py">
+      <provides>audio</provides>
+  </extension>
+  <extension point="xbmc.addon.metadata">
+      <summary lang="en">The Royal Concertgebouw Orchestra</summary>
```
Comment (enen92) [inline plugin.audio.rco/addon.xml]: default now is `en_gb`

File: plugin.audio.rco/addon.xml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+      <summary lang="nl">Het Koninklijk Concertgebouworkest</summary>
```
Comment (enen92) [inline plugin.audio.rco/addon.xml]: please also change this one to `nl_nl`

File: plugin.audio.rco/addon.xml [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+      <description lang="en">The Royal Concertgebouw Orchestra is a symphony orchestra that gives orchestral performances of the highest calibre in the world’s leading concert halls under the direction of the very best conductors. The activities it carries out in Amsterdam form the basis of its role as the Netherlands’ ambassador for international excellence. The Royal Concertgebouw Orchestra offers audiences emotional and intellectual enrichment, generating involvement and active loyalty.</description>
+      <description lang="nl">Het Koninklijk Concertgebouworkest is het symfonieorkest dat wereldwijd op de belangrijkste podia, onder leiding van de beste dirigenten en op het allerhoogste niveau uitvoeringen geeft van orkestmuziek. Zijn activiteiten in Amsterdam vormen de basis voor zijn rol als Nederlands ambassadeur van internationale excellentie. Het Concertgebouworkest biedt zijn publiek emotionele en intellectuele verrijking en verleidt tot betrokkenheid en actieve loyaliteit.</description>
+      <platform>all</platform>
+      <language>nl en</language>
```
Comment (enen92) [inline plugin.audio.rco/addon.xml]: language tag refers to media content and not to translations. Is the content (audio) this addon provides in english and dutch? If not please change to the correct value (probably just nl).

File: plugin.audio.rco/addon.xml [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+      <license>GPLv3</license>
+      <forum></forum>
+      <website>https://github.com/bosel/plugin.audio.rco</website>
+      <email></email>
+      <source>https://github.com/bosel/plugin.audio.rco</source>
+      <disclaimer lang="en">Copyright 2015 bosel.This program is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version. This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more details.</disclaimer>
```
Comment (enen92) [inline plugin.audio.rco/addon.xml]: `en_gb`

File: plugin.audio.rco/rco.py [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+import re
+import HTMLParser
+
+xbmc.log("plugin.video.rco:: Starting Addon")
+
+
+# magic; id of this plugin - cast to integer
+thisPlugin = int(sys.argv[1])
+
+settings = xbmcaddon.Addon(id='plugin.audio.rco')
```
Comment (enen92) [inline plugin.audio.rco/rco.py]: please remove the id kwargs. You can simply use `xbmcaddon.Addon()`.The id is only necessary when you try to access other addons from your addon.

File: plugin.audio.rco/rco.py [hunk 7]
```diff
@@ -0,0 +1,99 @@
...
+    @param list listing
+    @return void
+    """
+
+    # access global plugin id
+    global thisPlugin
+
+    # send each item to kodi
+    for item in listing:
+        listItem = xbmcgui.ListItem(item[0], thumbnailImage=os.path.join(IMG_DIR, "icon.png"))
```
Comment (enen92) [inline plugin.audio.rco/rco.py]: `thumbnailImage` is deprecated. Please use setArt instead:
`listItem.setArt({"thumb":os.path.join(IMG_DIR, "icon.png"), "fanart":os.path.join(IMG_DIR, "fanart.jpg")})`

File: plugin.audio.rco/rco.py [hunk 8]
```diff
@@ -0,0 +1,99 @@
...
+        listItem.setInfo(type="Music", infoLabels={"Title": item[0],
+                                                   "Artist": item[2],
+                                                   "Album": item[3],
+                                                   "Genre": "Classical",
+                                                   "Comment": item[0],
+                                                   "Year": item[len(item) - 1],
+                                                   # "AlbumArtist": item[4]
+                                                   })
+        listItem.setProperty("fanart_image", os.path.join(IMG_DIR, "fanart.jpg"))
```
Comment (enen92) [inline plugin.audio.rco/rco.py]: drop this one, you can set it in setArt as exemplified above

**Comment by enen92** [comment]
I'll create a second PR to mark the isengard version as broken. Once you adjust my comments, both will be merged.
Thank you

File: plugin.audio.rco/addon.xml [hunk 9]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.audio.rco" name="Het Koninklijk Concertgebouworkest" version="1.0.1" provider-name="bosel">
```
Comment (enen92) [inline plugin.audio.rco/addon.xml]: Please bump the version to 1.0.2 due to https://github.com/xbmc/repo-plugins/pull/973

**Comment by MartijnKaijser** [comment]
@bosel could you update according to the comments?

**Comment by bosel** [comment]
will do

**Comment by bosel** [comment]
This should do it:
https://github.com/xbmc/repo-plugins/pull/993

**Pull request closed without merging**